cumulusci.yml

```diff
@@ -37,6 +48,17 @@ tasks:
         options:
             path: unpackaged/config/dev_delete
 
+    deploy_einstein_templates:
+        group: 'EDA: Einstein Templates'
+        description: Deploys Einstein Analytics Templates
+        class_path: cumulusci.tasks.salesforce.Deploy
+        ui_options:
+            name: Deploy Einstein Analytics Templates
+        options:
+            path: unpackaged/config/analytics
+            namespace_inject: $project_config.project__package__namespace
+            unmanaged: False
```

Comment:
I saw at least one use of the hed__ prefix in the wave metadata. Should that be tokenized, or are we only trying to support use with managed EDA?

Reply:
Managed-only at this point.